### PR TITLE
ID-909 config switch to disable google

### PR DESCRIPTION
**Pull request by dvoet**
Ticket: https://broadworkbench.atlassian.net/browse/ID-909

---

**PR checklist**

- [ ] I've followed [the instructions](https://github.com/broadinstitute/sam/blob/develop/CONTRIBUTING.md#api-changes) if I've made any changes to the API, _especially_ if they're breaking changes
- [ ] I've filled out the [Security Risk Assessment](https://sdarq.dsp-appsec.broadinstitute.org/jira-ticket-risk-assesment) (requires Broad Internal network access) and attached the result to the JIRA ticket

### CODE REVIEW

File: src/main/scala/org/broadinstitute/dsde/workbench/sam/config/AppConfig.scala [hunk 1]
```diff
@@ -211,13 +211,17 @@ object AppConfig {
...
 
   def readConfig(config: Config): AppConfig = {
-    val googleConfigOption = for {
-      googleServices <- config.getAs[GoogleServicesConfig]("googleServices")
-    } yield GoogleConfig(
-      googleServices,
-      config.as[PetServiceAccountConfig]("petServiceAccount"),
-      config.as[Option[Duration]]("coordinatedAdminSdkBackoffDuration")
-    )
+    val googleEnabled = config.getAs[Boolean]("googleServices.googleEnabled").getOrElse(true)
```
Comment (tlangs) [inline src/main/scala/org/broadinstitute/dsde/workbench/sam/config/AppConfig.scala:214]: Why not just enable/disable Google based on the presence/absence of the `googleServices` config block?

Comment (dvoet) [reply src/main/scala/org/broadinstitute/dsde/workbench/sam/config/AppConfig.scala:214]: because we can't edit sam.conf inside the jar

**Review by tlangs — APPROVED** [review]
Approved, but with the knowledge that Sam will be in an insecure state at this point if GoogleServices are disabled.

File: src/main/scala/org/broadinstitute/dsde/workbench/sam/service/CloudExtensions.scala [hunk 2]
```diff
@@ -68,9 +68,9 @@ trait CloudExtensionsInitializer {
 }
 
 trait NoExtensions extends CloudExtensions {
-  override def isWorkbenchAdmin(memberEmail: WorkbenchEmail): Future[Boolean] = Future.successful(true)
+  override def isWorkbenchAdmin(memberEmail: WorkbenchEmail): Future[Boolean] = Future.successful(false)
 
-  override def isSamSuperAdmin(memberEmail: WorkbenchEmail): Future[Boolean] = Future.successful(true)
+  override def isSamSuperAdmin(memberEmail: WorkbenchEmail): Future[Boolean] = Future.successful(false)
```
Comment (dvoet) [inline src/main/scala/org/broadinstitute/dsde/workbench/sam/service/CloudExtensions.scala:73]: defaulting to true to allow everyone to be admin is not right

**Review by Ghost-in-a-Jar — APPROVED** [review]

**Pull request merged**